core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -625,10 +626,19 @@ private class SchedulerPool(name: String) extends LiveEntity {
 
 }
 
-private object LiveEntityHelpers {
+private[spark] object LiveEntityHelpers {
 
   private val stringInterner = Interners.newWeakInterner[String]()
 
+  private def accuValuetoString(value: Any): String = value match {
+    case list: java.util.List[_] =>
+      if (list.size() > 5) {
+        "[" + list.asScala.take(5).mkString(", ") + "...]"
```

Comment:
Can we make the output look like `truncatedString` just to make it look consistent?

Reply:
Yes, it is better to let users know how many more items there are.